client/sysinfo/src/sysinfo.rs

```diff
@@ -58,9 +166,9 @@ pub(crate) fn benchmark<E>(
 		}
 	}
 
-	let score = ((size * count) as f64 / elapsed.as_secs_f64()) / (1024.0 * 1024.0);
+	let score = Throughput::from_mibs((size * count) as f64 / elapsed.as_secs_f64());
```

Comment:
```suggestion
	let score = Throughput::from_kibs((size * count) as f64 / (elapsed.as_secs_f64() / 1024.0));
```
since it is returned in bytes/s.